### ISSUE TITLE
License to use

### ISSUE THREAD

**Issue by nooelan** [fabiodiluca/ExpressionSolver#1]
There is no mentioning of license for the ExpressionSolver in Nuget.org or GitHub. It would be good if it could list a license, e.g. MIT, so that users of this library may be sure about the terms the library may be used.

**Comment by maruthieee** [comment]
Yes. I would like to know the license type before using it in my application. @fabiodiluca please update us.
